pkg/env/nats_config.go

```diff
@@ -9,14 +9,10 @@ import (
 // NATSConfig represents the environment config for the Eventing Controller with Nats.
 type NATSConfig struct {
```

Comment:
Please remove `//go:build unit` (line 1) from `nats_config_test.go` file, fix the test `Test_GetNATSConfig`.